lib/av_client/aion_crypto.js

```diff
@@ -1076,38 +1076,35 @@ function addBigNums(bn1_hex, bn2_hex) {
   return sum_hex
 }
 
-module.exports = function() {
-  return {
-    'Curve': Curve,
-    'DiscreteLogarithmProof': DiscreteLogarithmProof,
-    'DiscreteLogarithmEqualityProof': DiscreteLogarithmEqualityProof,
-    'DiscreteLogarithmMultipleProof': DiscreteLogarithmMultipleProof,
-    'ElGamalScalarCryptogram': ElGamalScalarCryptogram,
-    'ElGamalPointCryptogram': ElGamalPointCryptogram,
-    'SchnorrSignature': SchnorrSignature,
-
-    'pointEquals': pointEquals,
-    'pointToBits': pointToBits,
-    'addPoints': addPoints,
-    'pointFromBits': pointFromBits,
-    'randomBN': randomBN,
-    'randomPoint': randomPoint,
-    'recoverYfromX': recoverYfromX,
-    'hashToBn': hashToBn,
-    'VOTE_ENCODING_TYPE': VOTE_ENCODING_TYPE,
-    'voteToPoint': voteToPoint,
-    'pointToVote': pointToVote,
-    'generateKeyPair': generateKeyPair,
-    'generateSchnorrSignature': generateSchnorrSignature,
-    'verifySchnorrSignature': verifySchnorrSignature,
-    'generateRandomNumber': generateRandomNumber,
-    'generateRandomPoint': generateRandomPoint,
-    'verifyEmptyCryptogramProof': verifyEmptyCryptogramProof,
-    'encryptVote': encryptVote,
-    'encryptVotePoint': encryptVotePoint,
-    'generateDiscreteLogarithmProof': generateDiscreteLogarithmProof,
-    'orderObjectByKeys': orderObjectByKeys,
-    'hashString': hashString,
-    'addBigNums': addBigNums,
-  }
+module.exports = {
+  Curve,
+  DiscreteLogarithmProof,
+  DiscreteLogarithmEqualityProof,
+  DiscreteLogarithmMultipleProof,
+  ElGamalScalarCryptogram,
+  ElGamalPointCryptogram,
+  SchnorrSignature,
+  //'pointEquals': pointEquals,                             // Not used?
+  //'pointToBits': pointToBits,                             // Not used?
+  //'addPoints': addPoints,                                 // Not used?
+  pointFromBits,
+  //'randomBN': randomBN,                                   // Not used?
+  //'randomPoint': randomPoint,                             // Not used?
+  //'recoverYfromX': recoverYfromX,                         // Not used?
+  //'hashToBn': hashToBn,                                   // Not used?
+  //'VOTE_ENCODING_TYPE': VOTE_ENCODING_TYPE,               // Not used?
+  // 'voteToPoint': voteToPoint,                            // Not used?
+  // 'pointToVote': pointToVote,                            // Not used?
+  generateKeyPair,
+  generateSchnorrSignature,
+  verifySchnorrSignature,
+  generateRandomNumber,
+  //'generateRandomPoint': generateRandomPoint,             // Not used?
+  verifyEmptyCryptogramProof,
+  encryptVote,
+  //'encryptVotePoint': encryptVotePoint,                   // Not used?
+  generateDiscreteLogarithmProof,
+  //'orderObjectByKeys': orderObjectByKeys,                 // Not used?
+  hashString,
+  //'addBigNums': addBigNums,                               // Not used?
```

Comment:
Coming back to these "Not used?" comments in https://github.com/aion-dk/js-client/pull/125